apps/web/src/components/FormList.tsx

```diff
@@ -18,21 +18,34 @@ import { FormInstanceEntity } from '@web/client';
 import { useState } from 'react';
 import { distance } from 'fastest-levenshtein';
 import { motion } from 'framer-motion';
+import Link from 'next/link';
+import { RightArrowIcon } from 'apps/web/src/static/icons';
+import { SearchAndSort } from 'apps/web/src/components/SearchAndRecent';
+import { ViewAll } from 'apps/web/src/components/ViewAll';
 
 /**
  * @param title - the title of the form list
  * @param formInstances - an array of form instances
  * @param color - the color of the form list
+ * @param extended - whether search and sort options are enabled
+ * @param link - link to page for category
+ * @param border - if true should have border
  * @returns a list of forms for the dashboard
  */
 export const FormList = ({
   title,
   formInstances,
   color,
+  extended,
+  link,
+  border,
 }: {
   title: string;
   formInstances: FormInstanceEntity[];
   color: string;
+  extended: boolean;
+  link: string;
+  border: boolean;
 }) => {
   const [searchQuery, setSearchQuery] = useState('');
```

Comment:
We want to pass this prop into `<SearchAndSort />` instead of declaring the hook in the component. We have `searchQuery` and `setSearchQuery` already declared here, so we don't need to declare it again in `<SearchAndSort />`. Instead, we can declare pass those values as props like `<SearchAndSort searchQuery={searchQuery} setSearchQuery={setSearchQuery} />` In SearchAndSort component, we can take those values like:

```
export const SearchAndSort = ({
  searchQuery,
  setSearchQuery,
}: {
  searchQuery: string;
  setSearchQuery: (searchQuery: string) => void;
}) => {
``` 
and then remove the useState in that component.